docs/pages/enterprise/sso/oidc.mdx

```diff
@@ -23,14 +23,37 @@ administrators to define policies like:
 ## Enable OIDC Authentication
 
 First, configure Teleport auth server to use OIDC authentication instead of the local
-user database. Update `/etc/teleport.yaml` as show below and restart the
-teleport daemon.
+user database. For self-hosted update `/etc/teleport.yaml` and restart the teleport daemon.  
+Use the Dynamic resources for Teleport Cloud and any other deployments.
 
-```yaml
-auth_service:
+<Tabs>
+  <TabItem label="Static Config (self-hosted)">
```

Comment:
Let's make the capitalization consistent between tab item labels: `Static config (self-hosted)` or `Static Config (Self-Hosted)`